helm/trento-ai-companion/values.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
This Helm chart is mostly autogenerated from some k8s manifests. This means it lacks several good practices around Helm charts... but it is a starting point.